t/11_hacluster.t

```diff
@@ -0,0 +1,57 @@
+use strict;
+use warnings;
+use Test::More;
+use Test::Exception;
+use Test::MockModule;
+use hacluster;
+use Scalar::Util 'looks_like_number';
+
+my %sbd_delay_params = (
+    'sbd_delay_start' => 'yes',
+    'corosync_token' => 5,
+    'corosync_consensus' => 5,
+    'sbd_watchdog_timeout' => 5,
+    'pcmk_delay_max' => 5
+);
+
+subtest '[calculate_sbd_start_delay] Check sbd_delay_start values' => sub {
+    my $sbd_delay;
+    my %value_vs_expected = (
+        'yes' => 55,
+        '1' => 55,
+        'no' => 0,
+        '0' => 0,
+        '120' => 120,
+    );
+
+    my $hacluster = Test::MockModule->new('hacluster', no_auto => 1);
+    $hacluster->redefine(record_info => sub { note(join(' ', 'RECORD_INFO -->', @_)); });
+    $hacluster->redefine(record_soft_failure => sub { note(join(' ', 'RECORD_SOFT_FAILURE -->', @_)); });
+    $hacluster->redefine(script_output => sub { note(join(' ', 'SCRIPT_OUTPUT -->', @_)); });
+
+    for my $input_value (keys %value_vs_expected) {
+        print "$input_value \n";
```

Comment:
I don't think this `print` is needed.

Reply:
Yes. Sorry... it was just a debug leftover. Removed.